src/main/cpp/subsystems/Telescope.cpp

```diff
@@ -6,24 +6,33 @@
 /*----------------------------------------------------------------------------*/
 
 #include "subsystems/Telescope.h"
+#include <frc/smartdashboard/SmartDashboard.h>
 
 Telescope::Telescope(
     const wpi::Twine& name,
     Components& components)
 : mComponents{components}
 {
     SetName(name);
-    mComponents.telescopeMotor.ConfigForwardSoftLimitEnable(true);
+    mComponents.telescopeMotor.ConfigForwardSoftLimitEnable(false);
+    mComponents.telescopeMotor.ConfigReverseSoftLimitEnable(true);
     mComponents.telescopeMotor.ConfigForwardSoftLimitThreshold(1300);
+    mComponents.telescopeMotor.ConfigReverseSoftLimitThreshold(0);
```

Comment:
Keep in mind that if the telescope is not down when the system reboots the lower threshold will be incorrect.